Source/EbsdLib/Utilities/ColorTable.cpp

```diff
@@ -102,8 +102,10 @@ void SIMPLColorTable::GetColorTable(int numColors, QVector<float>& colorsOut)
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
-std::vector<unsigned char> SIMPLColorTable::GetColorTable(size_t numColors, QJsonArray colorControlPoints)
+std::vector<unsigned char> SIMPLColorTable::GetColorTable(size_t numColors /*, QJsonArray colorControlPoints */)
```

Comment:
It would it be better to remove the method or class. This function does not do anything, but there is nothing stating that calling the method will not do what the programmer thinks it will. Any assumptions on functionality from methods that call this directly or indirectly are essentially broken.

Reply:
Method removed.